packages/storybook/package.json

```diff
@@ -29,6 +29,7 @@
   },
   "dependencies": {
     "@nrwl/cypress": "*",
+    "@nrwl/web": "*",
```

Comment:
Do we want to add dependency on `@nrwl/web` or just copy the parts needed from `webpack/entry`? I think angular workspaces probably won't need to have this package installed?